packages/venia-ui/i18n/en_US.json

```diff
@@ -362,5 +362,7 @@
     "wishlistPage.disabledMessage": "Sorry, this feature has been disabled.",
     "wishlistPage.fetchErrorMessage": "Something went wrong. Please refresh and try again.",
     "wishlistPage.headingText": "Favorites Lists",
-    "wishlistPage.wishlistDisabledMessage": "The wishlist is not currently available."
+    "wishlistPage.wishlistDisabledMessage": "The wishlist is not currently available.",
+    "orderDetails.waitingOnTracking": "Waiting for tracking information",
+    "orderItems.itemsHeading": "Items"
```

Comment:
Ideally we auto-generate as much as possible; relying on checkboxes is fragile, but fine for en_US for now. I do not think developers should be required to maintain the french pack, since it's entirely for testing purposes. Since we have the linting rule in place, I don't think that requirement would catch missed translations or add any value.